README.md

```diff
@@ -317,6 +317,23 @@ Content Steering features **not** supported:
 - PATHWAY-CLONES other replacements than HOST.
 
 
+## VR support
+Shaka Player supports VR when:
+- Content is automatically treated as VR if it fits the following criteria:
+  - HLS or DASH manifest
+  - fMP4 segments
+  - Init segment contains `prji` and `hfov` boxes
+- It is enabled via UI config.
```

Comment:
I'd rephrase this to make it clearer this is an alternative from the previous:

"Or, if it is manually enabled via the UI config."

Reply:
Done!